Sources/Testing/Testing.docc/DefiningTests.md

```diff
@@ -25,8 +25,9 @@ contains the test:
 import Testing
 ```
 
-- Note: Only import the testing library into a test target. Importing the
-  testing library into an application, library, or binary target isn't
+- Note: Only import the testing library into a test target or library meant for
+  test targets. Importing the testing library into a target intended for
+  distribution such as an application, app library, or executable target isn't
```

Comment:
Open to suggestions for better phrasing @iamleeg, but I hope my intent is clear!